tests/utils.py

```diff
@@ -195,9 +195,11 @@ def deposit_to_undeployed_sender(w3, entrypoint_contract, factory, factory_data)
     return sender
 
 
-def send_bundle_now(url=None):
+def send_bundle_now(w3):
+    # support 'pull' block building mode by sending a random 1 wei transaction
+    fund(w3, "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266", 1)
     try:
-        RPCRequest(method="debug_bundler_sendBundleNow").send(url)
+        RPCRequest(method="debug_bundler_sendBundleNow").send(None)
```

Comment:
don't break. the URL is used in p2p testing

Reply:
Where? I didn't see any place it was used. No test failed. Where are the P2P tests you are referring to?